README.md

```diff
@@ -1,2 +1,234 @@
-# js-ipid
-The IPID DID method implementation in JavaScript
+# did-ipid
```

Comment:
We should change the name of the repo according to this new name.

Reply:
I will change the repo name to `js-did-ipid`. Can you change every match of `ipfs-shipyard/js-ipid` to `ipfs-shipyard/js-did-ipid`.